ui/src/app/cohort-search/modifier-page/modifier-page.component.html

```diff
@@ -25,13 +25,13 @@
     <div *ngFor="let mod of modifiers; let i = index" [formGroupName]="mod.name">
       <div class="label-flex">
         <label class="modifier-event-header" [for]="mod.name">{{ mod.label }}</label>
```

Comment:
I'm able to add assas as a date and i don't get a validation message. I'm able to click Calculate and i get an error message